server/index.js

```diff
@@ -21,17 +53,35 @@ const openAIRoutes = require("./routes/openAI");
 const userProfileRoutes = require("./routes/userProfiles");
 
 // Use your authentication routes
-app.use("/api/auth", authRoutes);
+if(emailServiceEnabled){
```

Comment:
If email service is disabled, still the auth functionality should work, except that the frontend should not be calling the emailService.